presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -2,10 +2,23 @@
 JDBC Driver
 ===========
 
-Presto can be accessed from Java using the JDBC driver.
-Download :maven_download:`jdbc` and add it to the class path of your Java application.
+The Presto JDBC (Java Database Connectivity) driver allows users to access 
+Presto using Java-based applications. Both client- and server-side 
+applications, such as those used for reporting and database development, use 
+the JDBC driver. The JDBC driver is an API that provides methods to perform 
+database operations such as insert, update, delete and query.
 
-The driver is also available from Maven Central:
+Requirements
+------------
+
+The JDBC driver is compatible with Java 8, and can be used with applications 
```

Comment:
Maybe say "Java 8+" here so that it's more obvious without reading the second part of the sentence.